t/sanity.t

```diff
@@ -797,6 +810,7 @@ init_worker_by_lua '
 
             we.register(test_callback)
             we.post("content_by_lua","test_event")
+            ngx.sleep(2) -- wait for the polling interval
```

Comment:
do we need those intervals to be 2 seconds? can't we update the polling interval and reduce the wait? considering the number of occurrences, it is going to slow the tests down considerably

Reply:
Fixed, thanks!